kafka/4.6/modules/ROOT/pages/kafka-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,101 @@
+= Troubleshooting Apache Kafka Connector 4.6 - Mule 4
+
+To troubleshoot Anypoint Connector for Apache Kafka (Apache Kafka Connector), become familiar with the information about interpreting commonly thrown exception messages.
+
+== Understand Commonly Thrown Exceptions
+
+Here is a list of commonly thrown exception messages and how to interpret them:
+
+* KAFKA:ALREADY_COMMITTED
+
+ A commit fails because the commit is already performed.
+
+* KAFKA:AUTHENTICATION_ERROR(SECURITY)
+
+ An authentication exception is thrown.
+
+* KAFKA:AUTHORIZATION_ERROR(CLIENT_SECURITY)
+
+ An authorization exception is thrown.
+
+* KAFKA:COMMIT_FAILED
+
+ A commit fails with an unrecoverable error.
+
+* KAFKA:CONNECTIVITY
+
+ A problem occurred and a connection could not be established.
+
+* KAFKA:ILLEGAL_STATE
+
+ The consumer is not subscribed to any topics or manually assigned partitions to consume from.
+
+* KAFKA:INPUT_TOO_LARGE
+
+ The input is too large for the broker.
+
+* KAFKA:INVALID_ACK_MODE
+
+ An acknowledgement (ACK) or negative acknowledgement (NACK) operation is run over a consumer that is not in `MANUAL` mode.
+
+* KAFKA:INVALID_CONFIGURATION
+
+ An invalid configuration value is set for the consumer or the producer.
+
+* KAFKA:INVALID_CONNECTION
+
+ The referenced connection becomes invalid.
+
+* KAFKA:INVALID_INPUT
+
+ The parameters set for the operation are incorrect.
+
+* KAFKA:INVALID_OFFSET
+
+ The partition selected for the operation is less than `0`, is above the number of partitions, or is an invalid value.
+
+* KAFKA:INVALID_TOPIC
+
+ The topic selected for the operation is invalid and there is no automatic generation of said topic. This might also imply an invalid character in the topic name.
+
+* KAFKA:INVALID_TOPIC_PARTITION
+
+ The topic partition combination selected for the operation is invalid and there is no automatic generation of topics. This might also imply an invalid character in the topic name, or a non-existing partition.
+
+* KAFKA:NO_POLL_MADE
+
+ There are no inflight records available in the consumer (no poll was made).
```

Comment:
We should review this error in the connector. I am not sure if this message is correct.